### ISSUE TITLE
Please remove IP from blacklist

### ISSUE THREAD

**Issue by jsloyer** [docker/hub-feedback#1305]
It looks like Dockerhub is blacklisting the IP (168.1.201.140) for a gateway for one of my production Kubernetes clusters.  Can you please remove it from the blacklist?  This looks to be related to #298 and https://forums.docker.com/t/blacklisted-ip-address/43409.

```
curl --verbose --header 'Host: hub.
docker.com' 'http://54.156.233.45'                                              
* Rebuilt URL to: http://54.156.233.45/                                         
* Hostname was NOT found in DNS cache                                           
*   Trying 54.156.233.45...                                                     
* Connected to 54.156.233.45 (54.156.233.45) port 80 (#0)                       
> GET / HTTP/1.1                                                                
> User-Agent: curl/7.35.0                                                       
> Accept: */*                                                                   
> Host: hub.docker.com                                                          
>                                                                               
* HTTP 1.0, assume close after body                                             
< HTTP/1.0 500 Server Error                                                     
< Cache-Control: no-cache                                                       
< Connection: close                                                             
< Content-Type: text/html                                                       
<                                                                               
<html><body><h1>500 Server Error</h1>                                           
An internal server error occured.                                               
</body></html>                                                                  
                                                                                
* Closing connection 0 
```

**Comment by clarechawke** [comment]
The following IPs also seem to be blacklisted by DockerHub. Can they also be removed from the blacklist:

- 169.50.173.42
- 169.48.194.14
- 169.48.202.198
- 169.54.70.142
- 169.47.223.228

**Comment by binman-docker** [comment]
Hi @jsloyer @clarechawke - a number of SoftLayer-owned IPs were blocked due to abusive traffic pulling the httpd library image. We're working with Ralph Bateman at IBM to figure out what's going on.

For now, I've unblocked these IPs:
-169.47.223.228
-169.50.173.42
-168.1.201.140

These IPs were never included in the blocklist:
-169.48.194.14
-169.48.202.198
-169.54.70.142

**Comment by clarechawke** [comment]
Great. I tested the curls and these are now working on from my IPs.

**Comment by abhisan** [comment]
Hi, Looks like 52.73.2.136 is blacklisted. Can anyone remove it from the list. Thanks in advance.

**Comment by abhisan** [comment]
This is our production env NAT gateway elastic IP.

**Comment by binman-docker** [comment]
Hi @abhisan - can you open another issue to track so we don't clutter the inbox of other people on this older thread? 

In the meantime - our engineers had to block 52.73.2.136 in order to protect the service, because the IP was pulling (and is still trying to pull) alpine:latest over 500 times per minute.

Do you have some automation pulling that image that may be out of control? Once we see the requests come down to a reasonable level, we can unblock the IP.

If you actually need to pull images that often, I would suggest running your own local registry with a pull-through cache to Hub: https://docs.docker.com/registry/recipes/mirror/

**Comment by abhisan** [comment]
Thanks @binman-docker for your prompt response. We are moving to AWS Elastic Container Registry (ECR) for our docker image (java:8).  

Its seems, other team ( in our company ) is also using the same NAT gateway, and making this pull (alpine:latest) requests. We are investigating the issue, and will ask the concerned team to take the corrective measures. Thanks again.
